server/index.js

```diff
@@ -0,0 +1,137 @@
+#!/usr/bin / env node
+/*
+Copyright 2020 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+'use strict';
+
+/**
+ * Module dependencies.
+ */
+
+require('dotenv').config();
+
+var app = require('./app.js');
+var debug = require('debug')('server');
+var http = require('http');
+const open = require('open');
+const getPort = require('get-port');
+const crypto = require("crypto");
+
+let port;
+
+function run(portIncomming) {
+
+    if (isNaN(portIncomming) || portIncomming === undefined) {
```

Comment:
`isNan(undefined)` returns true, so you can actually remove `or` part of the condition. 
For details about isNan and when it returns what, see https://www.w3schools.com/jsref/jsref_isnan.asp.